Build/libHttpClient.Apple.C/libHttpClient.xcodeproj/project.pbxproj

```diff
@@ -720,11 +1019,26 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
+		D9EF888025A52685005C4BDF /* Resources */ = {
+			isa = PBXResourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+		D9FF0A9E25A536A20061B717 /* Resources */ = {
+			isa = PBXResourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
 		67EE8187236259D700A5F3DE /* Build openssl */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
>alwaysOutOfDate = 1; [](start = 3, length = 20)

What's the effect of this? Does this always rebuild then?

Reply:
It will always run the script, yeah. However, the script has logic in it to detect libs from a previous build and skip if they are present - so we want the script to run every time and bail if it detects it doesn't need to run.

Confusing, perhaps, but not sure I want to address this here beyond making the script run every time. Changed this because I had issues with the script not getting re-run for an incremental build when I needed it to be, plus the script is designed to handle this itself.